src/simple-cache-client.ts

```diff
@@ -252,51 +262,55 @@ export class SimpleCacheClient {
    * Add a value to the beginning of a list.
    * @param {string} cacheName - Name of the cache with the list.
    * @param {string} listName - The list to push to.
-   * @param {string | Uint8Array} - The value to push.
-   * @param {CollectionTtl} [ttl] - How the ttl should be managed. Defaults to the client's TTL.
-   * @param {number} [truncateFrontToSize] - If the list exceeds this length, remove excess from the start of the list. Must be positive.
+   * @param {string | Uint8Array} value - The value to push.
+   * @param {FrontTruncatableCallOptions} [options]
+   * @param {number} [options.truncateFrontToSize] - If the list exceeds this
+   * length, remove excess from the start of the list. Must be positive.
+   * @param {CollectionTtl} [options.ttl] - How the TTL should be managed.
+   * Defaults to the client's TTL.
    * @return {Promise<CacheListPushBack.Response>}
    */
   public async listPushBack(
     cacheName: string,
     listName: string,
     value: string | Uint8Array,
-    ttl?: CollectionTtl,
-    truncateFrontToSize?: number
+    options?: FrontTruncatableCallOptions
   ): Promise<CacheListPushBack.Response> {
     const client = this.getNextDataClient();
     return await client.listPushBack(
       cacheName,
       listName,
       value,
-      ttl,
-      truncateFrontToSize
+      options?.truncateFrontToSize,
```

Comment:
I'm curious why you didn't pass the options through here. When the options grow, this signature is going to grow as well. Just a thought.

Reply:
I wanted to get the first version up so people could see the shape of the options object. I am not opposed to passing it through if that is cleaner.

Reply:
I'm going to keep the internal api as is for now. It won't be set in stone with the 1.0 release since it is internal. I added a ticket to revisit this in the future. [https://github.com/momentohq/client-sdk-nodejs/issues/240](url)